tabpy-server/server_tests/test_config.py

```diff
@@ -1,52 +1,52 @@
+import os
 import unittest
 try:
     from unittest.mock import patch, call
 except ImportError:
     from mock import patch, call
 
 from os.path import isfile
-from os import remove
 from sys import version_info
 
-from tabpy import get_config
+from tabpy_server.tabpy import get_config
 
 
 class TestConfigEnvironmentCalls(unittest.TestCase):
 
-    def file_exists(fn):
+    def file_exists(fn):  # Hmm - reinventing the wheel here?
         if fn == './common/config.py':
             return False
         return True
 
-    @patch('tabpy.TabPyState')
-    @patch('tabpy._get_state_from_file')
-    @patch('tabpy.shutil')
-    @patch('tabpy.PythonServiceHandler')
-    @patch('tabpy.os.path.exists', return_value=True)
-    @patch('tabpy.os.path.isfile', side_effect=file_exists)
-    @patch('tabpy.os')
+    @patch('tabpy_server.tabpy.TabPyState')
```

Comment:
Didn't look in detail but off the top of my head this patching machinery is convoluted and may not be delivering the intended effects. Probably a good separate PR if anyone is interested